### ISSUE TITLE
truffle.js vs truffle-config.js

### ISSUE THREAD

**Issue by Quazia** [aragon/aragon-cli#110]
On Windows machines, it's often necessary to rename the truffle config file ``truffle-config.js`` so as to avoid confusion with truffle (the tool). There are [other solutions](http://truffleframework.com/docs/advanced/configuration#resolving-naming-conflicts-on-windows) but when going with the name change route the following is output:
```
[20:07:36] Start a local Ethereum network [started]
[20:07:36] Start a local Ethereum network [failed]
[20:07:36] → Cannot read property 'connection' of undefined
 × Cannot read property 'connection' of undefined
```

I believe it is the following line that becomes problematic:
https://github.com/aragon/aragon-cli/blob/a43747cc1e70ddc371640e9f9ab349b3725149d0/src/helpers/truffle-config.js#L6

suggested solution would be something along the lines of creating an additional else-if statement to handle ``truffle-config.js``.

If another solution is used (such as removing the .JS extension from PATHEXT environment variable) ``aragon run`` can successfully start the network so this isn't a critical issue.

**Comment by izqui** [comment]
#105 adds support for ‘truffle-config.js’ files :)

**Comment by Quazia** [comment]
Silly me I should have known from that amazing PR description 👍

**Comment by izqui** [comment]
LOL sorry @Quazia. Closed by #105
